owl/SOMA-ACT.owl

```diff
@@ -196,6 +198,13 @@ Declaration(NamedIndividual(SOMA:ExecutionState_Succeeded))
 #   Object Properties
 ############################
 
+# Object Property: SOMA:affects (affects)
+
+AnnotationAssertion(rdfs:comment SOMA:affects "Simple relationship between two actions to express that a variation in the course or outcome of the the object (the affectee) would have resulted in a variation in the subject (the affector), e.g., a planning task that sets parameters such as goal position affects the subsequently executed pick-and-place task that uses that parameter.")
```

Comment:
Apologies for the nitpick but it seems the comment should say, a change in the subject (affector) results in a change of the object (affectee). Otherwise the PR is good to go!

Reply:
Fixed this and the comments of its inverse!